pandas/_libs/tslibs/parsing.pyx

```diff
@@ -467,8 +473,14 @@ cdef inline object _parse_dateabbr_string(object date_string, object default,
     raise ValueError(f'Unable to parse {date_string}')
 
 
-cdef dateutil_parse(str timestr, object default, ignoretz=False,
-                    tzinfos=None, dayfirst=None, yearfirst=None):
+cdef dateutil_parse(
+    str timestr,
+    object default,
+    bint ignoretz=False,
+    tzinfos=None,
+    dayfirst=None,
```

Comment:
bint no?

Reply:
Doing so will require changing the default value of ```dayfirst```, is it okay?

Reply:
yes please.  ping on green